python/ray/serve/_private/router.py

```diff
@@ -292,7 +292,7 @@ async def _resolve_deployment_responses(
 
     async def schedule_and_send_request(
         self, pr: PendingRequest
-    ) -> Tuple[Union[ray.ObjectRef, "ray._raylet.ObjectRefGenerator"], str]:
+    ) -> Tuple[Union[ray.ObjectRef, ray.ObjectRefGenerator], str]:
```

Comment:
To check my understanding: so `schedule_and_send_request` will always return `ray.ObjectRefGenerator` if we've turned on the feature flag for the new routing? And this is handled by the new `_should_resolve_gen_to_obj_ref`?